src/fondant/pipeline.py

```diff
@@ -153,6 +155,132 @@ def from_registry(
             ephemeral_storage_size=ephemeral_storage_size,
         )
 
+    @staticmethod
+    def get_image_manifest(image_ref: str):
+        """Retrieve the Docker image manifest.
+
+        Args:
+            image_ref: The Docker image reference (e.g., 'registry/image:tag').
+
+        Returns:
+            dict: The parsed JSON manifest.
+        """
+        cmd = ["docker", "manifest", "inspect", image_ref]
```

Comment:
This will make docker a hard requirement up until now it was more of a soft requirement for local running. If we do this we might want to use the docker python sdk

Reply:
yeah indeed, I'm also not really happy with this but I don't think it's possible [link]. (https://stackoverflow.com/questions/56178911/how-to-obtain-docker-image-digest-from-tag).
For now the SDK seems to only support inspecting local images [link](https://docker-py.readthedocs.io/en/stable/api.html#:~:text=inspect_container(container)%C2%B6). The `docker manifest inspect` seems to a [new command](https://docs.docker.com/engine/reference/commandline/manifest_inspect/) so it might not have been integrated yet with the SDK. 

Perhaps we can use `curl` but then it requires to know the url of different registries [link](https://cloud.google.com/kubernetes-engine/docs/archive/using-container-images)